include/illixr/cpu_timer.hpp

```diff
@@ -159,35 +160,29 @@ class should_profile_class {
 public:
     should_profile_class() {
         const char* ILLIXR_STDOUT_METRICS = getenv("ILLIXR_STDOUT_METRICS");
-        actually_should_profile           = ILLIXR_STDOUT_METRICS && (strcmp(ILLIXR_STDOUT_METRICS, "y") == 0);
+        actually_should_profile_          = ILLIXR_STDOUT_METRICS && (strcmp(ILLIXR_STDOUT_METRICS, "y") == 0);
```

Comment:
Nit: should we consider renaming this private variable to `should_profile_bool_`? Not important, this comment can be ignored.

Reply:
should_profile_bool_ seems redundant as it names the type in the variable. we should probably leave it as is for now